src/client/Client.cc

```diff
@@ -5209,32 +5249,32 @@ int Client::get_or_create(Inode *dir, const char* name,
   return 0;
 }
 
-int Client::path_walk(const filepath& origpath, Inode **final, bool followsym)
+int Client::path_walk(const filepath& origpath, C_InodeRef *end, bool followsym)
```

Comment:
On Mon, Jun 1, 2015 at 6:07 AM, Gregory Farnum <notifications@github.com>
wrote:

> In src/client/Client.cc
> <https://github.com/ceph/ceph/pull/4753#discussion_r31394841>:
>
> > @@ -5209,32 +5249,32 @@ int Client::get_or_create(Inode *dir, const char* name,
> >    return 0;
> >  }
> >
> > -int Client::path_walk(const filepath& origpath, Inode **final, bool followsym)
> > +int Client::path_walk(const filepath& origpath, C_InodeRef *end, bool followsym)
>
> Why the name change?
>

final is keyword. it looks weired in editor


> --
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph/pull/4753/files#r31394841>.
>